.github/workflows/ci.yml

```diff
@@ -102,7 +105,7 @@ jobs:
           packagePath: results/Documentation/renderer
 
       - name: Upload test results as artifacts
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         if: always()
```

Comment:
the last step is again checking out the repo, what is useless so please remove

Reply:
I tried to remove, but without that it doesn't work, because of some post jobs

Reply:
can you show me a workflow where this happened? It does not make sense to me...

Reply:
I can't find it now, but I could push changes that you could see

Reply:
the problem is that the release documentation action is checked out in the same repo, the linter action is. This checkout most probably is overwriting the linter-action. Please change this (e.g. change the checkout path)

Reply:
You are right, I changed it.